src/stores/executionStore.ts

```diff
@@ -426,6 +439,14 @@ export const useExecutionStore = defineStore('execution', () => {
      * The error from the previous execution.
      */
     lastExecutionError,
+    /**
+     * NodeLocatorId for the most recent execution error.
+     */
+    lastExecutionErrorNodeLocatorId,
+    /**
+     * Local node ID for the most recent execution error.
+     */
+    lastExecutionErrorNodeId,
```

Comment:
The names are pretty descriptive, not sure how much the comments add.

I'm a little annoyed that there are two forms of ID, but I'm sure there's a reason.